pkg/cmd/roachtest/roachtestutil/mixedversion/mutators.go

```diff
@@ -426,13 +430,21 @@ func (m panicNodeMutator) Generate(
 
 		isIncompatibleStep := func(s *singleStep) bool {
 			// Restarting the system on a different node while our panicked node is still dead can
-			// cause the cluster to lose quorum, so we avoid any system restarts.
-			_, restart := s.impl.(restartWithNewBinaryStep)
+			// cause the cluster to lose quorum, so we avoid any system restarts. If
+			// the cluster has a high enough node count however, we can upreplicate
+			// to 5X before panicking, allowing us to safely restart other nodes.
+			restartImpl, restart := s.impl.(restartWithNewBinaryStep)
+			if supportsUpReplication {
+				// We can restart other nodes, but we do not want to restart the
+				// node that is being panicked, as this can cause conflict with the
```

Comment:
nit: `as the panic recover step expects the node to be down in order to restart it`. Just to iterate what the actual issue is and to clarify its a simplification we make in the framework rather than an actual crdb limitation.